gef.py

```diff
@@ -4522,9 +4522,27 @@ def stop(self):
 
 
 #
-# Commands
+# Context Panes
 #
 
+def register_external_context_pane(pane_name, display_pane_function, pane_title_function):
+    """
+    Registering function for new GEF Context View. pane_title_function must return a string.
+    pane_name: a string that has no spaces (used in settings)
+    display_pane_function: a function that uses gef_print() to print strings
+    pane_title_function: a function that returns a string, which will be displayed as the title
+
+    Example Usage:
+    def display_pane(): gef_print("Wow, I am a context pane!")
+    def pane_title(): return "example:pane"
+    register_external_context_pane("example_pane", display_pane, pane_title)
+    """
+    return __gef__.add_context_pane(pane_name, display_pane_function, pane_title_function)
```

Comment:
Why do we care what this returns? If we do, that's fine, add a mention to the doc string: `Returns true if successful` or something.